### ISSUE TITLE
No longer displaying icons on the right side of Steamgifts?

### ISSUE THREAD

**Issue by phrostb** [Revadike/SteamWebIntegration#80]
Hey there.. so I noticed that the old SWI userscript was no longer displaying the icons on the right side of Steamgifts, so I tried the Firefox extension instead. That one, also isn't displaying them. I managed to see just one. I posted screenshots below:

![image](https://github.com/user-attachments/assets/77bd635a-72c5-47e0-956b-55383ed89bfa)

![image](https://github.com/user-attachments/assets/0840381d-b8d1-411c-8f8d-11259b48fca3)

What do you think?

**Comment by Revadike** [comment]
@phrostb try this option
![image](https://github.com/user-attachments/assets/9badef03-b2d9-461d-9aed-7b70b19b1ffd)

**Comment by phrostb** [comment]
@Revadike doesn't seem to have helped. is it working properly for you?

**Comment by Revadike** [comment]
yeah, those buttons are disabled if you're on that dynamic content mode

**Comment by phrostb** [comment]
ok please let me know if you need anything from me (logs or whatever)

**Comment by Revadike** [comment]
![image](https://github.com/user-attachments/assets/1c8d5359-c43b-45fb-9779-ef93e1eb643e)

Working fine for me in the latest version
